### ISSUE TITLE
Wording

### ISSUE THREAD

**Issue by bits-of-static** [lblissett/dn-discuss#10]
The current wording for the frontpage proposed by Johannes needs review.

Subtitle:
> DiscourseNet (DN) is an open network for anyone active in the field of discourse studies. Through our website and our activities, we aim to create an open, multilingual, inclusive and non-hierarchical space. This website allows its users to connect with a large community of discourse scholars about discourse-related CFP's, events and publications. You may also create groups to facilitate communication and create a personal profile to increase your visibility in the discourse community.

The text needs to fit into the given space otherwise it will not work. Long text does not animate people into reading it and it does not integrate into the design.

From content, it should talk about what it is and not what it offers. What is on offer, is intended to be presented in the section below. Those three texts (unique selling points) must be of approximately same length. The title and the text should work well together. The third one currently repeats itself.

** Connect with your colleagues **
> Connect and collaborate with discourse researchers from around the world in all scientific disciplines.

** Read and discuss publications **
> Find the latest research and connect with the authors.

** Make your work visible **
> Share your work with others.

**Comment by lblissett** [comment]
We dont have a working about text at the moment: http://drupalize-da.de/en/about is filled with wrong information. Thext was just used as a filler. This about page needs an complete rewrite.

What peter was refering is the text at the frontpage. You have a big headline, a subtitle and three smaller texts under the symbols. 

So we need a shorter subtitle than the one proposed from Johannes (which we have not included at the dev site now, but peter posted it above). 

And we need texts under the symbols which has a similar length. You can check this at the current dev webpage and you will see it looks strange, if the variation of lenght is too long. 

Please check this site: https://www.researchgate.net/ to get an idea of a good example (and also, to check that we are not too close with our texts to them).

**Comment by bits-of-static** [comment]
In a coupleof hous, you can look at thewording I propose:

**DiscourseNet: International Association for Discourse Studies**  
> We are an open network for anyone active in the field of discourse studies. We aim to create an open, multilingual, inclusive and non-hierarchical space.

**Connect with your colleagues**  
>Connect and collaborate with discourse researchers from around the world in all scientific disciplines.

**Connect with your colleagues**  
> We collect not yet published papers - find cutting-edge research and connect with the authors.

**Make your work visible**  
> Create groups to facilitate communication and create a personal profile to increase your visibility in the community.

**Comment by lblissett** [comment]
I just made a new update to our last version. You can see the new proposal of the wording on our test site online.

**Comment by SinghJN** [comment]
Great, thanks. I feel this works. To my taste, it sounds a bit corporate, as if we provide a service to the discourse research community. But perhaps, our Board members will have comments about this. You can leave it as it is for now. We might ask for changes at a later stage.

Generally, for the English version, please take out all instances of the word 'scientific'. In Anglophone countries this means 'natural sciences' (naturwissenschaftlich) and it is not an accurate translation of German 'wissenschaftlich', which refers to both Geisteswissenschaften and Naturwissenschaften. Can you replace 'scientific' with 'academic' please? Thanks

**Comment by lblissett** [comment]
Yes, thanks this is a very valuable remark. You should feel comfortable with the wording we used and it is probably more geared into corporate style than the old page. 

Good, we gonna change this to academic.

**Comment by jangermuller** [comment]
I agree and perhaps let’s not sound too corporate!

**Comment by jangermuller** [comment]
What about the following presentation?

The Discourse Studies platform
by DiscourseNet 

We create an open, multilingual, inclusive and non-hierarchical space for anyone active in the field of Discourse Studies. 

Connect

Collaborate with discourse researchers from around the world in all disciplines.

Publish

Disseminate events, find research, share work. 

Join

Create a profile, join a group and become part of the community.

**Comment by bits-of-static** [comment]
I find it looks great. Satisfied?

![dn front wording proposal](https://user-images.githubusercontent.com/13779752/46127196-364c7800-c230-11e8-8087-f9104f105f71.JPG)

**Comment by jangermuller** [comment]
Looks great, yes! A scroll-down invitation (so that people realise there is more below) is the only thing still missing!
